frontend/src/constants/Constants.js

```diff
@@ -110,19 +112,114 @@ export const INTERMEDIATE_API_ALL_PRIVATE_PROJECTS_ENDPOINT =
+export const VS30_MAP_IMG_ALT = "Vs30 Map";
+export const EPSILON_DISAGG_PLOT_ALT = "Epsilon disagg plot";
+export const SOURCE_DISAGG_PLOT_ALT = "Source disagg plot";
+
+// Hazard Curve labels
+export const FAULT = "Fault";
+export const BRANCHES = "Branches";
+export const ENSEMBLE_MEAN = "Ensemble mean";
+export const TOTAL = "Total";
+export const NZS1170P5 = "NZS1170.5";
+export const NZTA = "NZTA";
+export const ANNUAL_RATE_OF_EXCEEDANCE = "Annual rate of exceedance";
+export const LOWER_PERCENTILE = "16<sup>th</sup> percentile";
+export const UPPER_PERCENTILE = "84<sup>th</sup> percentile";
+export const ROTD_FIFTY = "RotD50";
+
+// Disaggregation labels
+export const MEAN_MAGNITUDE = "Mean Magnitude";
+export const MEAN_RRUP = "Mean Rrup";
+export const MEAN_EPSILON = "Mean Epsilon";
+export const NAME = "Name";
+export const CONTRIBUTION = "Contribution";
+export const ANNUAL_RECURRENCE_RATE = "Annual recurrence rate";
+export const MAGNITUDE = "Magnitude";
+export const RRUP = "Rrup";
+export const SHOW_MORE = "Show More...";
+export const SHOW_LESS = "Show Less...";
+
+// GMS labels
+export const PSEUDO_ACCELERATION_RESPONSE_SPECTRA =
+  "Pseudo acceleration response spectra";
+export const LOWER_AND_UPPER_BOUND_LIMITS = "Lower and upper bound limits";
+export const REFERENCE_POINT = "Reference Point";
+export const CUMULATIVE_PROB_CDF = "Cumulative Probability, CDF";
+export const SHORTEN_SELECTED_GM = "Selected GMs";
+export const SELECTED_GM = "Selected Ground Motions";
+export const REALISATIONS = "Realisations";
+export const GCIM = "GCIM";
+export const SPECTRAL_ACCELERATION = "Spectral acceleration";
+export const BOUNDS = "Bounds";
+export const RUPTURE_DISTANCE = "Rupture distance";
+export const AVAILABLE_GROUND_MOTIONS = "Available ground motions";
+export const DISAGGREGATION_DSITRIBUTION = "Disaggregation distribution";
+export const SITE_SPECIFIC = "Site-specific";
+export const GET_CAUSAL_PARAMS_BOUNDS = "Get causal parameters bounds";
+export const GET_IM_VECTOR_WEIGHTS = "Get IM vector weights";
+
+// Select's placeholders
+export const PLACEHOLDER_NOT_AVAILABLE = "Not available";
+export const PLACEHOLDER_SELECT_SIGN = "Select...";
+export const PLACEHOLDER_LOADING = "Loading...";
+export const PLACESHOLDER_SELECT_PROJECT_ID =
+  "Please select the Project ID first...";
+export const PLACEHOLDER_SELECT_LOCATION =
+  "Please select the Location first...";
+export const PLACEHOLDER_SELECT_VS30 = "Please select the Vs30 first...";
+export const PLACEHOLDER_SELECT_IM =
+  "Please select the Intensity Measure first...";
+export const PLACEHOLDER_SELECT_PERIOD = "Select period...";
+export const PLACEHOLDER_NOT_APPLICABLE = "N/A";
+export const PLACEHOLDER_SELECT_COMPONENT = "Select component...";
+
+// Units in labels
+export const SECONDS = "(s)";
+export const KILOMETRE = "(km)";
+export const METRE_PER_SECOND = "(m/s)";
+export const YEARS = "(years)";
+export const PERCENTAGE = "(%)";
+export const G_FORCE = "(g)";
```

Comment:
I would have these with a suffix of _UNIT (I presume these compile out for the webapp side itself?)

Reply:
Yeah, these will be with a suffix of _UNIT as you pointed out. (In upcoming PR)
I just realized that in the non-projects tab with Mw, a symbol for Magnitude, because `MAGNITUDE` is already reserved. Hence, I had to add a suffix, and might as well add this suffix to other units to keep the consistency :+1: